appveyor.yml

```diff
@@ -6,12 +6,22 @@ before_build:
   - nuget restore nunit.sln
 
 build_script: 
-  - msbuild nunit.sln /p:Configuration=Release /t:Rebuild
+  - ps: |
+      .\appveyor.ps1
```

Comment:
If I read the docs correctly our only options for scripting are a batch file or powershell. Is that right?